frontend/src/components/StreamingBox.vue

```diff
@@ -147,28 +162,26 @@ export default {
 
   data() {
     return {
-      stickyStreamingSrc: null,
-      isVideoAvailable: false,
-      isVideoVisible: false,
-      remainingSecondsCurrentVideoCycle: 30,
-      remainingSecondsUntilNextCycle: -1,
-      currentBitrate: null,
+      STREAM_STATUS,
+      streamStatus: STREAM_STATUS.Unavailable,
+      isTrackMuted: false,
+      isVideoLoading: false,
+
       slowLinkLoss: 0,
-      slowLinkShowing: false, // show on mousenter
-      slowLinkHiding: false, // hide on moseleave
-      trackMuted: false,
-      videoLoading: false,
-      printerStockImgSrc: '#svg-3d-printer'
+      slowLinkLossLimit: 50,
```

Comment:
Define limit here in single place. Also we may want to use it in other places in the future